packages/babel-plugin-html-attributes-to-jsx/package.json

```diff
@@ -1,4 +1,5 @@
 {
+  "private": true,
```

Comment:
why is this a private package?

Reply:
because it’s no longer used. I propose actually deprecating them / removing them